pkg/messaging/v1/invoke_method_response.go

```diff
@@ -35,7 +33,7 @@ func NewInvokeMethodResponse(statusCode int32, statusMessage string, statusDetai
 func InternalInvokeResponse(resp *internalv1pb.InternalInvokeResponse) (*InvokeMethodResponse, error) {
```

Comment:
why is error needed ? can it be removed?

Reply:
This method uses like below (there is another place). Please see the line of `return`. Otherwise, we need to add if and 
 
https://github.com/dapr/dapr/blob/baed8143e2251a6fd2095925308e9c8b5ec1cfe8/pkg/messaging/direct_messaging.go#L147-L166

And wanted to keep consistent with InvokeRequest method signature.

https://github.com/dapr/dapr/blob/8da72e6b346de0de54e14e70398407b78490542f/pkg/messaging/v1/invoke_method_request.go#L52-L60

Let me know if you think it doesn't make sense.

Reply:
Here if it does not return an error, I do not see a point a maintaining consistency with another method. @yaron2 thoughts?

Reply:
InternalInvokeRequest and InternalInvokeResponse has similar function signature.  I want to keep them consistent. 

func InternalInvokeRequest(pb *internalv1pb.InternalInvokeRequest) (*InvokeMethodRequest, error)
func InternalInvokeResponse(resp *internalv1pb.InternalInvokeResponse) (*InvokeMethodResponse, error)

No harm to have error in this case.